### DISCUSSION TITLE
Login possible with web auth tokens?

### DISCUSSION THREAD

**Discussion by mattstg3** [tweepy/tweepy#2158] [Questions] [answered]
### Question

Hello, I want to create a tool using tweepy but I have a question. Is it possible to login with tweepy using the web auth tokens instead of oauth? If yes, how exactly does it work? 
A twitter web auth token looks like this: 05e721b6218c86a9edf9666fe30c0ffe66e41d43
Thanks in advance!

### Checklist

- [ ] I have searched for duplicate questions.
- [ ] If applicable, I have removed any visible credentials from my code and/or screenshots.

**Comment by Harmon758** [comment, marked as answer]
No, Tweepy uses Twitter's developer API, which requires authentication with a developer App.
https://developer.twitter.com/en/docs/authentication/overview
